src/main/java/de/unistuttgart/iste/gits/quiz_service/persistence/dao/NumericQuestionEntity.java

```diff
@@ -0,0 +1,31 @@
+package de.unistuttgart.iste.gits.quiz_service.persistence.dao;
+
+import de.unistuttgart.iste.gits.common.resource_markdown.ResourceMarkdownEmbeddable;
+import de.unistuttgart.iste.gits.common.resource_markdown.ResourceMarkdownEntity;
+import jakarta.persistence.*;
+import lombok.*;
+import lombok.experimental.SuperBuilder;
+
+@Entity(name = "NumericQuestion")
+@ToString(callSuper = true)
+@Getter
+@Setter
+@EqualsAndHashCode(callSuper = true)
+@SuperBuilder
+@AllArgsConstructor
+@NoArgsConstructor
+public class NumericQuestionEntity extends QuestionEntity {
```

Comment:
I thought we said this question type is unnecessary because you can just use an ExactAnswerQuestion. I can see that you have implemented a tolerance so it makes a bit more sense I guess but the question remains if it's really worth it to spend time on this.

Reply:
but with the exact answer question, if I say the correct answer is "1" and the users enters "1.0" it would be wrong

Reply:
true